src/html5/tokenizer/token.rs

```diff
@@ -232,4 +232,36 @@ mod tests {
         let token = Token::Eof;
         assert_eq!(format!("{}", token), "EOF");
     }
+    #[test]
+    fn test_is_start_tag() {
+        let token = Token::StartTag{
+            name: "div".to_string(),
+            is_self_closing: false,
+            attributes: HashMap::new(),
+        };
+        assert!(token.is_start_tag("div"));
+        assert!(!token.is_start_tag("span"));
+    }
+    #[test]
+    fn test_is_any_start_tag() {
```

Comment:
Can we add a newline between each test and the one that follows?

```suggestion
    }

    #[test]
    fn test_is_any_start_tag() {
```

Reply:
ohk i did that